### ISSUE TITLE
Invalid environments can be brought up and can't be deleted

### ISSUE THREAD

**Issue by erwinvaneyk** [fission/fission#181]
Currently, you are able to bring up environments, with images that do not exist or that do not conform with the interface of fission environments. Understandably, you can't check this completely without actually downloading and trying to run the environment, but it might be an idea to at least to have a check for non-existing environments.

For example: `fission env create --name foobar --image fission/foobar` results in 3 failing generic pods:
```
fission-function   foobar-dd5c5f9b...   1/2       ImagePullBackOff   0          5m       
fission-function   foobar-dd5c5f9b...   1/2       ImagePullBackOff   0          5m       
fission-function   foobar-dd5c5f9b...   1/2       ImagePullBackOff   0          5m       
``` 

The second issue occurs when trying to correct the issue. When trying to delete the environment (`fission env delete foobar`), the following error
`Failed to delete environment: Delete failed: 404 page not found`

**Comment by soamvasani** [comment]
Yes we really ought to validat better.  But the delete should have worked; that looks like a separate bug.

**Comment by erwinvaneyk** [comment]
You are right. The second issue seemed to be a quirk. It does work now as expected 👍

**Comment by life1347** [comment]
related to https://github.com/fission/fission/issues/368

**Comment by sanketsudake** [comment]
This has been inactive for 1653 days. We're scrubbing the backlog and closing stale items to keep it actionable. If this is still relevant, please reopen or comment and we'll pick it back up.
